images/manageiq-base/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN mkdir build && \
 FROM registry.access.redhat.com/ubi8/ubi
 MAINTAINER ManageIQ https://manageiq.org
 
-ARG ARCH=x86_64
+ARG ARCH=$(uname -m)
```

Comment:
This will not work - ARCH should be passed from the caller as an ARG if it's needed

Reply:
Added a `RUN echo $ARCH` right after this and I get

```
#5 [stage-1  2/12] RUN echo $(uname
#5 sha256:d7be9006ff8aec6f61879ceb483f7d0e3c9cd732e46e8de4ff9c9fe198688a53
#5 0.220 $(uname
#5 DONE 0.2s
```